models/business-unit/src/company/generator.ts

```diff
@@ -30,6 +30,7 @@ const generator = Generator<TCompany>({
     key: fake((f) => f.lorem.slug(2)),
     status: oneOf(...Object.values(status)),
     stores: [],
+    inheritedStores: [],
```

Comment:
This was missing so the model didn't include this field even though the SDK has it.  the presence of this field is conditional which is the problem but including it here will save us some headache when testing